edgar_tool/cli.py

```diff
@@ -46,15 +48,23 @@ def _validate_text_search_args(
         destination.lower().endswith(ext) for ext in SUPPORTED_OUTPUT_EXTENSIONS
     ):
         raise ValueError(
-            f"Destination file must have one of the following extensions: {', '.join(SUPPORTED_OUTPUT_EXTENSIONS)}"
+            f"Destination file must have one of the following extensions: {'; '.join(SUPPORTED_OUTPUT_EXTENSIONS)}"
         )
     if (
-        filing_type
-        and filing_type.lower() not in TEXT_SEARCH_FILING_CATEGORIES_MAPPING.keys()
+        filing_form
+        and filing_form not in TEXT_SEARCH_FILING_VS_MAPPING_CATEGORIES_MAPPING.keys()
     ):
         raise ValueError(
-            f"Filing type must be one of: {', '.join(TEXT_SEARCH_FILING_CATEGORIES_MAPPING.keys())}"
-        )
+            f"Filing form group must be one of: {'; '.join(TEXT_SEARCH_FILING_VS_MAPPING_CATEGORIES_MAPPING.keys())}"
+    )
+    if single_forms:
+        single_list = [item for sublist in TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.values() for item in
+                       sublist]
+        invalid_forms = [form for form in single_forms if form not in single_list]
+        if invalid_forms:
+            raise ValueError(
+                f"Single forms must be one or more of: {single_list}"
+            )
```

Comment:
here it does not raise error as before: it handles cases where only `single_forms` are present, those forms just needs to be one of all of the possible values of `TEXT_SEARCH_CATEGORY_FORM_GROUPINGS`